devicetypes/smartthings/zooz-4-in-1-sensor.src/zooz-4-in-1-sensor.groovy

```diff
@@ -23,8 +23,12 @@ metadata {
 		capability "Health Check"
 		capability "Tamper Alert"
 
-		fingerprint mfr: "027A", prod: "2021", model: "2101", deviceJoinName: "Zooz Multipurpose Sensor" //Zooz 4-in-1 sensor
-		fingerprint mfr: "0109", prod: "2021", model: "2101", deviceJoinName: "Vision Multipurpose Sensor" //ZP3111US 4-in-1 Motion
+		fingerprint mfr: "027A", prod: "2021", model: "2101", deviceJoinName: "Zooz Multipurpose Sensor"
+		//Zooz 4-in-1 sensor
```

Comment:
Let's keep the comments indicating the device on the same line as the fingerprint.

Reply:
Fixed